### ISSUE TITLE
Logits support

### ISSUE THREAD

**Issue by ogencoglu** [rankseg/rankseg#3]
Thanks for the interesting work.

Can rankseg be used with raw logits instead of probabilities? Unbounded values instead of [0, 1] bounds essentially. Typically in the range [-10, 10] let's say.

**Comment by ogencoglu** [comment]
Btw back then I got confused with the docs which mentioned logits:

> Works with ANY pre-trained logit/prob-outcome segmentation model without retraining
